server/app.ts

```diff
@@ -85,6 +89,39 @@ export let start = async function(port, hostname) {
 
   winston.info('Configuration ready; launching HTTP server');
 
+  if (config.cron.type === 'onbox') {
+    // run cron-like scheduler
+    const strategicUpdater = new CronJob(
+      config.cron.cronTime,
+      () => {
+        runStrategic(config, pg());
+      },
+      false,
+      config.cron.timeZone
+    );
+    strategicUpdater.start();
+  } else {
+    // output crontab execution file
+    fs.writeFile(
+      'strategicScheduler',
+      config.cron.cronTime + ' /server/util/strategicLogger.js',
+      err => {
```

Comment:
If you think callback error handling is painful (I do) then you may want to check out `util.promisify` introduced in Node 8: http://2ality.com/2017/05/util-promisify.html